userspace/falco/configuration.h

```diff
@@ -37,6 +37,16 @@ limitations under the License.
 #include "event_drops.h"
 #include "falco_outputs.h"
 
+enum class engine_kind_t : uint8_t
+{
+	KMOD,
+	EBPF,
+	MODERN_EBPF,
+	REPLAY,
+	GVISOR,
+	NONE
```

Comment:
Do you know if this is a particular reason for this ordering?

Not saying it is an issue, just I don't get it.

Reply:
No special reason!